### PR TITLE
Upgrade spc module and large scale test

### PR DESCRIPTION
**Pull request by OSsunNO**
- Changes:
1. Implement replacing emoji with ':explain_text:'
2. Modify README.md
3. Change the code that replaces special characters into the code that removes the special characters
4. Conduct large scale test
- TODO:
1. Discuss options related to the scope of implementation of the SPC module
2. Define the dictionary according to the selected option
3. Implement additional features of spc module

### CODE REVIEW

**Comment by OSsunNO** [comment]
- Large scale dataset
'dataset.xlsx' is on the Google Drive due to uploading issue

**Comment by OSsunNO** [comment]
SPC module implementation options
- opt 1: eliminating all the emojis and special characters
- opt 2: replace the emojis and remove all the special characters
- opt 3: replace all the emojis and special character combinations (1-digit special character is not included)
According to the selected option, the dictionary for spc module will be changed.

**Comment by OSsunNO** [comment]
Execution Results
- input
Untitled
- output
Untitled
- running time
Untitled

**Comment by OSsunNO** [comment]
Execution results 1
- input
image
- output
image
- running time
image
Execution results 2
- input
image
- output
image
- running time
image

File: dt-demo-be/modules/spc/run.py [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+import emoji
+
 
 def run(fileName):
     
     print("---Special character replacing module processing---")
     
     # Read Excel and save to ../database/fake.txt
-    test_dict = pd.read_csv("../database/seoul_freeboard.csv")
+    test_data = pd.read_excel("../database/dataset.xlsx")
```
Comment (yoouung) [inline dt-demo-be/modules/spc/run.py:12]: can you leave a comment with the format of this dataset so that we can test it for ourselves?

**Review by yoouung — APPROVED** [review]
thanks for working on holidays.... :)

**Comment by yoouung** [comment]
can you update the second comment with the link of the dataset?
i.e.) [dataset link](https://drive.google.com/file/d/1FWw9_3jCIQJIUG-rGskSE6AwqjA6ZAGg/view?usp=drive_link)

**Comment by OSsunNO** [comment]
> can you update the second comment with the link of the dataset? i.e.) [dataset link](https://drive.google.com/file/d/1FWw9_3jCIQJIUG-rGskSE6AwqjA6ZAGg/view?usp=drive_link)

I changed the second comment :)
Thank you for your feedback!

**Pull request merged**